docs/migration/seidb_archive_migration.md

```diff
@@ -0,0 +1,150 @@
+# SeiDB Archive Migration Guide
+
+## Overview
+SeiDB is the next generation of chain storage in SeiV2.
+One issue for running SeiDB on archive nodes is that we need to keep the full state of the chain, so we can't
+state sync it and clear out previous iavl data.
+
+In order to run an archive node with SeiDB, we need to run a migration from iavl to sei-db.
+
+The overall process will work as follows:
+
+1. Stop archive node and note down its height, call it MIGRATION_HEIGHT
+2. Update config to enable SeiDB (state committment + state store)
+3. Run sc migration at the MIGRATION_HEIGHT
+4. Re start seid with `--migrate-iavl` enabled (migrating state store in background)
+5. Verify migration at various sampled heights once state store is complete
+6. Stop seid, clear out iavl and restart seid normally, now only using SeiDB fully
```

Comment:
We should not delete IAVL data here but launch with SeiDB first, and only delete the IAVL data after we successfully run the node for a while

Reply:
Updated